### PR TITLE
Feature/algorithms go

### PR DESCRIPTION
**Pull request by Prashanth6361**

### CODE REVIEW

File: stackGo/stack/stack.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	}
+}
+
+func (s *Stack) Push(v interface{}) error {
+	l := len(s.value)
+	if l < s.size {
+		s.value = append(s.value, v)
+		return nil
+	}
+	return errors.New("Full stack")
```
Comment (rajkagrawal) [inline stackGo/stack/stack.go]: error checking should be done first  instea of l < s.size see if s.size == s.value then return full stack

File: queueGo/queue/queue.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+func New(size int) *Queue {
+	return &Queue{
+		size:  size,
+		value: make([]interface{}, 0),
+	}
+}
+
+func (a *Queue) Enqueue(v interface{}) error {
+	l := a.Length()
```
Comment (rajkagrawal) [inline queueGo/queue/queue.go:18]: error checking first

**Review by rajkagrawal — CHANGES_REQUESTED** [review]

File: stackGo/stack/stack.go (hunk 1 above)
Comment (Prashanth6361) [reply stackGo/stack/stack.go]: I have updated sir

File: queueGo/queue/queue.go (hunk 2 above)
Comment (Prashanth6361) [reply queueGo/queue/queue.go:18]: I have updated sir

**Pull request merged**